sysprep/activate_instance.ps1

```diff
@@ -273,29 +273,28 @@ function Verify-ActivationStatus {
       $slmgr_status = & cscript //E:VBScript //nologo $env:windir\system32\slmgr.vbs /dli
     }
     catch {
-      return $active
+      $active = $false
```

Comment:
The variable is already set to false and only a positive result would set it true.

I'd suggest the catch simply log the issue.

Reply:
I changed it to write to host.